commit d7f113e02043bfa21e6bfd5e7617973eef2ecf33 (refs/changes/87/2887/2)
Author: Kody A Kantor <kody.kantor@gmail.com>
Date:   2017-11-01T22:05:42+00:00 (1 year, 11 months ago)
    
    MORAY-447 strange latency data point in reported moray metrics
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/README.md b/README.md
index 41929a7..96e6810 100644
--- a/README.md
+++ b/README.md
@@ -251,6 +251,9 @@ Each of the metrics returned include the following metadata labels:
 - Zone UUID
 - PID
 
+The request latency and request counter metrics additionally include a metadata
+label denoting the RPC method used (e.g. 'getbucket').
+
 The metric collection functionality is intended to be consumed by a monitoring
 service like a Prometheus or InfluxDB server.
 
diff --git a/package.json b/package.json
index 9ef41db..f40c3b6 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "main": "lib/index.js",
     "dependencies": {
         "ajv": "4.11.4",
-        "artedi": "1.1.0",
+        "artedi": "1.1.1",
         "assert-plus": "1.0.0",
         "bunyan": "1.8.10",
         "bunyan-syslog": "0.2.2",
@@ -15,7 +15,7 @@
         "crc": "0.2.1",
         "dtrace-provider": "~0.8",
         "deep-equal": "0.0.0",
-        "fast": "2.3.0",
+        "fast": "2.3.1",
         "ip6addr": "0.1.1",
         "jsprim": "1.4.0",
         "kang": "1.2.0",
